internal/runner/runner.go

```diff
@@ -114,11 +114,11 @@ func (r *CommandRunner) runStep(ctx context.Context, step *CommandStep, dry bool
 	if err != nil {
 		return nil, err
 	}
-	posixEnv = append(posixEnv, "PWD="+pwd)
+	// posixEnv = append(posixEnv, "PWD="+pwd)
```

Comment:
Alternatively we could set a basic PATH statically.
```suggestion
	posixEnv = append(posixEnv, "PWD="+pwd)
	posixEnv = append(posixEnv, "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin")
```